drake/systems/plants/eiquadprog.h

```diff
@@ -619,4 +619,4 @@ inline void delete_constraint(MatrixXd& R, MatrixXd& J, VectorXi& A, VectorXd& u
 
 }
 
-#endif
+#endif  //  DRAKE_SYSTEMS_PLANTS_EIQUADPROG_H_
```

Comment:
DG: Right, will fix.
LF: For these PRs, I'm not running clang-format, just cpplint.